packages/libxml2.rb

```diff
@@ -22,7 +22,6 @@ class Libxml2 < Package
     x86_64: 'ebb3b81155231be702617246a78945a3698142f63b3138c5763b9813d57dbe19'
   })
 
-  depends_on 'gcc'
```

Comment:
Add `depends_on 'gcc::libs'`?

Reply:
This is a deleted line (`libxml2` does not depend on `gcc`)